mucommander-core/src/main/java/com/mucommander/ui/tabs/HideableTabbedPane.java

```diff
@@ -188,7 +188,10 @@ protected void addAndSelectTab(T tab) {
 	 * @param updater - object that will be used to update the tab
 	 */
 	protected void updateCurrentTab(TabUpdater<T> updater) {
-		tabsCollection.updateTab(getSelectedIndex(), updater);
+		var idx = getSelectedIndex();
+		if (idx >= 0) {
+			tabsCollection.updateTab(idx, updater);
+		}
```

Comment:
I've managed to reproduce it by:
1. stopping both threads that initialize the folder panels in the constructor of FileTableTabs
2. let the thread of the right folder panel continue and stop it in LocationChanger#tryChangeCurrentFolderInternal, after `mainFrame.setNoEventsMode(true);` is called
3. let the thread of the left folder panel continue

the thread of the left folder panel would raise the same exception
the explanation is that in no-event-mode, the selected index of FileTableTabbedPane is not set (see FileTableTabbedPane#setSelectedIndex) and therefore we get -1 from getSelectedIndex here

this change would solve that issue but the selected index won't be set properly
I'd prefer to see how we can avoid this situation

Reply:
posted an alternative solution in #1201